proposal/string-split.md

```diff
@@ -0,0 +1,110 @@
+# Add split function to Strings Module: Draft 1
+
+*([Issue](https://github.com/sass/sass/issues/1950))*
+
+This proposal adds `string.split()` to the `sass:string` module.
+
+## Table of Contents
+
+* [Background](#background)
+* [Summary](#summary)
+* [Semantics](#semantics)
+  * [`string.split()`](#string.split)
+
+## Background
+
+> This section is non-normative.
+
+The `sass:string` module contains several functions for
+manipulating and finding out information about `strings`.
+Currently, though, there is no built-in function that splits
+one `string` into a list of substrings, and authors have been
+creating their own versions of functions that achieve this
+functionality.
+
+## Summary
+
+> This section is non-normative.
+
+This proposal adds the `string.split()` function to the
+`sass:string` module. The function takes a `string`, splits it
+based on a provided `separator`, and returns a space-separated
+list of substrings.
+
+This could be used to take a provided `string` and repurpose
+parts of it for some other use. For example, fonts 
+contained in a font stack list could be split off and 
+then used as keys in a new map. 
+
+Examples:
+
+  - ```scss
+    $fonts: "Helvetica Neue, Helvetica, Arial";
+    string.split($fonts, ', '); // "Helvetica Neue" "Helvetica" "Arial"
+    ```
+
+A third argument can limit the number of `strings` 
+returned in the list:
+
+- ```scss
+  string.split($fonts, ', ', 2); // "Helvetica Neue" "Helvetica"
+  ```
+
+
+An empty `$separator` returns all characters in the original `string`:
```

Comment:
I suggest adding explicitly that this splits by Unicode codepoints (as that's what sass strings count as characters), not by grapheme clusters or by UTF-16 codepoints (Javascripts splits by codepoints as its indexes are also based on codepoints)